### DISCUSSION TITLE
Microfon and speaker

### DISCUSSION THREAD

**Discussion by icho40** [roleoroleo/yi-hack-Allwinner-v2#756] [Q&A]
Hello, 
I have 3 cameras running with yi-hack-Allwinner-v2.
None of them make any sound or record via microphone.
Is this not supported?
According to 
[https://github.com/roleoroleo/yi-hack-Allwinner-v2/tree/master#features](url)
there should b a sound- channel.
I tried that with vlc-player, but didn't work (see attached file)
[debug.txt](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/13447811/debug.txt)

**Comment by roleoroleo** [comment]
Yes there is a sound channel.
When you enable sound in the webgui the rtsp stream contains an audio channel.
About the speaker you can use it in different modes:
- web services
- ha integration
- command line
